components/asset/introduction.rst

```diff
@@ -0,0 +1,288 @@
+.. index::
+   single: Asset
+   single: Components; Asset
+
+The Asset Component
+===================
+
+   The Asset component manages URL generation and versioning of web assets such
+   as CSS stylesheets, JavaScript files and image files.
+
+In the past, it was common for web applications to hardcode the URLs of the web
+assets. For example:
+
+.. code-block:: html
+
+    <link rel="stylesheet" type="text/css" href="/css/main.css">
+
+    <!-- ... -->
+
+    <a href="/"><img src="/images/logo.png"></a>
+
+This practice is no longer recommended unless the web application is extremely
+simple. The main problems of hardcoding asset URLs are the following:
```

Comment:
`This can be a disadvantage because:` and then removing the because below to make them sentences stating the disadvantage.